src/Test.UnitTests.Sarif/Visitors/SuppressVisitorTests.cs

```diff
@@ -125,12 +125,12 @@ public void SuppressVisitor_ShouldFlowPropertiesCorrectly()
 
                 if (timestamps && suppression.TryGetProperty("timeUtc", out DateTime timeUtc))
                 {
-                    timeUtc.Should().BeCloseTo(DateTime.UtcNow);
+                    timeUtc.Should().BeCloseTo(DateTime.UtcNow, 500);
```

Comment:
>500 [](http://example.com/codeflow?start=64&length=3)

this change to fix random error in existing tests.
the default time is only 20ms and if the server not fast enough, it will be above that.